composer.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "leverage-php/lambda-skeleton",
+    "license": "MIT",
+    "autoload": {
+        "psr-4": {
+            "App\\": "src/",
+            "App\\Test\\": "test/"
```

Comment:
DISCUSS: I've been using App and Test as the namespaces rather than App and App\Test.

Any reason we need to add the App part of the Test namespace?

Reply:
No, I think it's just because I'm used to seeing it this way.